tools/integration_tests/concurrent_operations/setup_test.go

```diff
@@ -47,7 +47,7 @@ func TestMain(m *testing.M) {
 
 	// Create common storage client to be used in test.
 	ctx = context.Background()
-	closeStorageClient := client.CreateStorageClientWithTimeOut(&ctx, &storageClient, time.Minute*15)
+	closeStorageClient := client.CreateStorageClientWithTimeOut(&ctx, &storageClient, time.Minute*30)
```

Comment:
why is this increased?

Reply:
FYI, we want to get rid of this timeout: https://github.com/GoogleCloudPlatform/gcsfuse/pull/2385